lib/base/RequestClient.js

```diff
@@ -94,6 +95,15 @@ RequestClient.prototype.request = function(opts) {
   this.lastRequest = new Request(optionsRequest);
 
   axios(options).then((response) => {
+    if (opts.logLevel === "debug") {
+      parseRequest(optionsRequest)
```

Comment:
Should log the actual request (just `options`) and do this _before_ making the request in case of weird errors n such.